stdpopsim/cli.py

```diff
@@ -587,7 +590,11 @@ def write_simulation_summary(engine, model, contig, samples, seed=None):
     # Get information about relevant contig
     gmap = "None" if contig.genetic_map is None else contig.genetic_map.id
     mean_recomb_rate = contig.recombination_map.mean_rate
-    mut_rate = contig.mutation_rate
+    # use the model mutation rate, if provided with the demographic model
+    if model.mutation_rate is not None:
```

Comment:
probably good to add a logging.info() saying where the mutation rate is coming from

Reply:
I'm not sure if what I have now here is exactly what you mean... let me know if you had something else in mind.

Reply:
could be good to print the actual rate in that logging message so the user can see it